ibis/backends/tests/test_set_ops.py

```diff
@@ -115,6 +122,8 @@ def test_intersect(backend, alltypes, df, distinct):
         param(True, id="distinct"),
     ],
 )
+# Question: why doesn't it get green without strict=False? (smelly)
```

Comment:
@cpcloud do you have any insights on this? If not I can dig deeper on this later on (follow up PRs).